### PR TITLE
Add TransferAccessMessage.toBytes()

### PR DESCRIPTION
**Pull request by alextaka**

### CODE REVIEW

File: u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java [hunk 1]
```diff
@@ -102,6 +105,40 @@ public static TransferAccessMessage fromBytes(byte[] data) throws U2FException {
   }
   
   /**
+   * Write a TransferAccessMessage into a raw byte array.
```
Comment (leshi) [inline u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java]: Alternate suggested description:
`Return TransferAccessMessage as a byte array`

File: u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java [hunk 2]
```diff
@@ -102,6 +105,40 @@ public static TransferAccessMessage fromBytes(byte[] data) throws U2FException {
...
+   * @throws U2FException 
+   */
+  public byte[] toBytes() throws U2FException {
+    int sizeOfSequenceNumber = 1;
+    int lengthField_signatureUsingAuthenticationKey = 1;
```
Comment (leshi) [inline u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java]: don't use underscores and camel case.

use came case for member variable names.

Also, it's sizeOfLengthField, right?  "sizeof" is important I think

**Review by leshi — CHANGES_REQUESTED** [review]

File: u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java (hunk 2 above)
Comment (alextaka) [reply u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java]: done.

Changed to:
sizeOfLengthFieldForSignatureUsingAuthenticationKey 
sizeOfLengthFieldForSignatureUsingAttestationKey

File: u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java (hunk 1 above)
Comment (alextaka) [reply u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java]: done.

**Comment by leshi** [comment]
LGTM

**Review by leshi — APPROVED** [review]

**Pull request merged**